packages/e2e/test/local-network/local-network.test.ts

```diff
@@ -29,6 +31,55 @@ export const env = envalid.cleanEnv(process.env, {
 
 const logger = getLogger(env.LOGGER_MIN_SEVERITY);
 
+const getWalletStateSnapshot = async (wallet: ObservableWallet) => {
```

Comment:
We could export these utils from `util.ts` to be used by both this and delegation test

Reply:
This should be no longer required since using `TxBuilder` makes the test no longer require `getWalletStateSnapshot`.
Initially I did it, but being no longer required I rolled it back to make the change having an impact as small as possible.